emission/net/api/visualize.py

```diff
@@ -29,36 +35,46 @@ def carbon_by_zip(start,end):
             carbon_list.append(tempdict)
     return {"weightedLoc": carbon_list}
 
-def Berkeley_pop_route(start,end):
-    Sections = get_section_db()
-    list_of_point=[]
-    # print(berkeley_area())
-    for section in Sections.find({"$and":[{'In_UCB':True },{'type':'move'},{"section_start_datetime": {"$gte": start, "$lt": end}}]}):
-        for pnt in section['track_points']:
-                list_of_point.append(pnt['track_location']['coordinates'])
-    return {"latlng": list_of_point}
+def Berkeley_pop_route(start_dt, end_dt):
+    box = [ [-122.267443, 37.864693], [-122.250985, 37.880687] ]
+    tl = esdt.get_aggregate_timeline_from_dt_box(start_dt, end_dt, box)
+    gj_list = gfc.get_geojson_for_timeline_viz(None, tl)
+    list_of_points=[]
+    for gj in gj_list:
+        for feature in gj:
+            if feature['type'] == 'FeatureCollection':
+                for feat in feature['features']:
+
+                    if "properties" not in feat:
+                        continue
+                if feat['properties']['feature_type'] == "section":
+                    points = feat['geometry']['coordinates']
+                    list_of_point.extend(points)
+    return {"latlng": list_of_points}
 
-def Commute_pop_route(modeId,start,end):
-    Sections = get_section_db()
-    list_of_point=[]
-    # print(berkeley_area())
-    commuteQuery = {"$or": [{'commute': 'to'}, {'commute': 'from'}]}
-    modeQuery = {"$or": [{'mode': modeId}, getConfirmationModeQuery(modeId)]}
-    dateTimeQuery = {"section_start_datetime": {"$gte": start, "$lt": end}}
-#   findQuery = {"$and":[modeQuery,dateTimeQuery,{'type':'move'}]}
-    findQuery = {"$and":[modeQuery,{'type':'move'}]}
-    logging.debug("About to execute query %s" % findQuery)
-    findQuery = {}
-    for section in Sections.find({"$and":[modeQuery,dateTimeQuery,{'type':'move'}]}):
-        if len(section['track_points']) > 5:
-          # skip routes that have less than 3 points
-          for pnt in section['track_points'][5:-5]:
-                  list_of_point.append(pnt['track_location']['coordinates'])
 
+def Commute_pop_route(mode, start_dt, end_dt):
+    tl = esdt.get_aggregate_timeline_from_dt(start_dt, end_dt)
+    gj_list = gfc.get_geojson_for_timeline_viz(None, tl)
+ 
+    logging.debug("len gl list is %d" % len(gj_list))
+    list_of_point=[]
+ 
+    for gj in gj_list:
+        print "hrre"
+        logging.debug("Found %d sections in the trip" % len(gj))
```

Comment:
It turned out that these were not sections, right? So maybe change the log statement to make sense?